shared/django_apps/dummy_settings.py

```diff
@@ -8,18 +8,63 @@
 
 # Install apps so that you can make migrations for them
 INSTALLED_APPS = [
+    "shared.django_apps.legacy_migrations",
     "shared.django_apps.pg_telemetry",
     "shared.django_apps.ts_telemetry",
     "shared.django_apps.rollouts",
+    # API models
+    "django.contrib.admin",
+    "django.contrib.auth",
+    "django.contrib.contenttypes",
+    "django.contrib.messages",
+    "django.contrib.postgres",
```

Comment:
why were these necessary again? weren't they looped in because of some helper function that used a template or something? can we get rid of them?

Reply:
admin -> `LogEntry` used in codecov_auth helper
contentypes -> get_gitlab_url used in codecov_auth helper
postgres -> I get test errors otherwise, `syntax error at or near "gin_trgm_ops"`, https://stackoverflow.com/questions/37334902/postgres-syntax-error-creating-index-using-prgm-gin

The rest I've deleted and I'll test tmrw